src/Extractor.ts

```diff
@@ -1334,6 +1392,11 @@ class Extractor {
       // This is fine, we just don't know what it is. This should be the expected
       // case for operation types such as `Query`, `Mutation`, and `Subscription`
       // where there is not strong convention around.
+    } else if (
+      node.type.kind === ts.SyntaxKind.TypeReference &&
```

Comment:
What is the user experience if they use `@gqlExternal` but it's not a type reference? Does it just silently ignore the `@gqlExternal`? If so, can we avoid that with a custom error message for that case?